src/svgdevicecontext.cpp

```diff
@@ -339,7 +317,8 @@ void SvgDeviceContext::StartTextGraphic(Object *object, std::string gClass, std:
 
 void SvgDeviceContext::ResumeGraphic(Object *object, std::string gId)
 {
-    std::string xpath = "//g[@id=\"" + gId + "\"]";
```

Comment:
Found the problem.

Reply:
Maybe, but before to implement a change it would be good to evaluate how expensive it is. It seems the the xpath implementation is pretty efficient. Having a map with every single object is not going to be much better, especially since the map keys are going to be string ids.